test/event/handlers/CommandParserHandlerTest.ts

```diff
@@ -107,6 +107,17 @@ describe("CommandParserHandler", () => {
 			expect(errorWasThrown).to.be.false;
 		});
 
+		it("should not run command if whitelisted to a channel which is not the execution channel", async () => {
+			const message = BaseMocks.getMessage();
+			const command = new MockWhitelistedCommand();
+			const commandSpy = sandbox.spy(command, "run");
+
+			message.channel.id = "abcdef";
```

Comment:
We want to avoid modifying properties of `BaseMock` as they're static - realise this is probably taken from the above test but we should change that one too